minio.yaml

```diff
@@ -2,14 +2,19 @@ apiVersion: template.openshift.io/v1
 kind: Template
 labels:
   template: Minio
-message: Minio template
 metadata:
   annotations:
-    description: Deploys Minio using existing PVCs
+    description: This template deploys S3 compatible private Object Store Minio using existing PVCs.
+    tags: S3, Object Store
+    openshift.io/display-name: Minio Object Store
+    iconClass: icon-scala
+    openshift.io/provider-display-name: CSC-IT Center for Science Ltd.
+    openshift.io/documentation-url: https://github.com/CSCfi/Minio-OpenShift/blob/master/README.md
+    openshift.io/support-url: https://rahti.csc.fi/
 
-    openshift.io/display-name: minio
-    openshift.io/provider-display-name: CSC
   name: minio
+  message: In case you want to use existing PVC for Minio Storage, please update PVC name with the PVC you want to use.
```

Comment:
```suggestion
  message: In case you want to use existing PVC for Minio Storage, please update PVC name with the PVC you want to use. In this case, Rahti will issue an error that no new PVC is created but proceeds with creating the rest of the application nevertheless.
```